pkg/refproto/ssl_game_event.proto

```diff
@@ -219,6 +225,8 @@ message GameEvent {
         optional uint32 by_bot = 2;
         // the location of the bot
         optional Location location = 3;
+        // the location of the ball at the moment when this foul occurred
+        optional Location ball_location = 4;
```

Comment:
Although it also has no disadvantage, this field will never be populated as the message is not sent by the autorefs

Reply:
well, same for the location. I'd say, add it for consistency.